accepted/9999-npm-audit-assertions.md

```diff
@@ -0,0 +1,64 @@
+# `npm audit` assertions
+
+## Summary
+
+A mechanism for module maintainers to assert that their modules are not impacted by advisories.
+
+```
+npm auidit assert --module=<module-name> --id=<advisory identifier> --impactful=<boolean>
```

Comment:
minor typo in the `npm audit` command here

Reply:
```suggestion
npm audit assert --module=<module-name> --id=<advisory identifier> --impactful=<boolean>
```